src/Widgets/calendar/CalendarModel.vala

```diff
@@ -22,49 +22,55 @@
  *  it under the terms of the GNU General Public License as published by
  *  the Free Software Foundation, either version 3 of the License, or
  *  (at your option) any later version.
- * 
+ *
  *  This program is distributed in the hope that it will be useful,
  *  but WITHOUT ANY WARRANTY; without even the implied warranty of
  *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  *  GNU General Public License for more details.
- * 
+ *
  *  You should have received a copy of the GNU General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
- * 
+ *
  */
 namespace DateTime.Widgets {
     public class Event : GLib.Object {
         public GLib.DateTime date;
-        Util.DateRange range;
+        public Util.DateRange range;
+        public E.Source source;
+
         public string summary;
         public bool day_event = false;
         public bool alarm = false;
         public GLib.DateTime start_time;
         public GLib.DateTime end_time;
+        public unowned iCal.Component ical;
+        public E.SourceCalendar? cal;
 
-        public Event (GLib.DateTime date, Util.DateRange range, iCal.Component comp) {
+        public Event (GLib.DateTime date, Util.DateRange range, iCal.Component ical, E.Source source) {
             this.date = date;
             this.range = range;
-            this.summary = comp.get_summary ();
+            this.source = source;
+            this.summary = ical.get_summary ();
 
-            Util.get_local_datetimes_from_icalcomponent (comp, out start_time, out end_time);
+            Util.get_local_datetimes_from_icalcomponent (ical, out start_time, out end_time);
             if (end_time == null) {
                 alarm = true;
             } else if (Util.is_the_all_day (start_time, end_time)) {
                 day_event = true;
-                return;
             }
+
+            this.cal = (E.SourceCalendar?)source.get_extension (E.SOURCE_EXTENSION_CALENDAR);
         }
 
         public string get_label () {
             if (day_event) {
-                return summary;
+                return "%s\n%s".printf (summary, _("All Day"));
```

Comment:
Please revert all these string changes for this branch. This doesn't have anything to do with the event dots

Reply:
They're from the Events Pane PR, was a mis-copypaste, fixing!